privaterelay/templates/includes/dashboard-filter.html

```diff
@@ -0,0 +1,59 @@
+{% load static %}
+{% load ftl %}
+{% ftlconf bundle='privaterelay.ftl_bundles.main' %}
+
+<div class="c-filter">
+    <div class="c-filter-search">
+        <div class="c-filter-icon">
+            <img src="/static/images/icon-search.svg" alt="">
+        </div>
+        <form action="">
+            <input class="c-filter-search-input" type="search" name="search" id="search">
+        </form>
+    </div>
+    <div class="c-filter-date">
+        <div class="c-filter-icon">
+            <img src="/static/images/icon-calendar.svg" alt="">
+        </div>
+
+    </div>
+    <div class="c-filter-cateogry">
+        <div class="c-filter-icon">
+            <img src="/static/images/icon-filter.svg" alt="">
+        </div>
+    </div>
+    <div class="c-filter-action">
+        <!-- Button -->
+        <form action="/emails/" class="dash-create" method="POST">
```

Comment:
Ooo, and let's fix this action to `{% url  'emails-index' %}` tag.

Reply:
Ended up moving these to be displayed via CSS to account for hover-state colors. We also could import the actual SVG, rather than the image tag. Not sure the best way to manage that. (Would at least only require one SVG file per icon instead of two)